Makefile

```diff
@@ -0,0 +1,5 @@
+build-dev:
+	docker-compose build
```

Comment:
- Use `--parallel` for faster builds. 
- Add a command for installing node-deps locally. 
```
install-local-dependencies:
	npm --prefix ./frontend/ install
```